plugins/base/build.gradle.kts

```diff
@@ -4,6 +4,8 @@ plugins {
     id("com.jfrog.bintray")
 }
 
+val testUtils by configurations.creating
```

Comment:
See #1055 for discussion regarding this idea!

Reply:
Yes, i took it from that PR with hope that it is a good idea xd Once this PR will be resolved i'll rebase my changes

Reply:
Ok, i've rebased this PR on recent dev changes to get rid to this configuration